coordinateutils/src/HealpixSkyMap.cxx

```diff
@@ -1034,9 +1072,13 @@ PYBINDINGS("coordinateutils")
 	    .def(bp::init<>())
 	    .add_property("nside", &HealpixSkyMap::nside)
 	    .add_property("nested", &HealpixSkyMap::nested)
+	    .add_property("shift_ra", &HealpixSkyMap::IsShiftRa, HealpixSkyMap_setshiftra,
+		"True if the ringsparse representation of the map is stored "
+		"with the rings centered at ra = 0 deg, rather than ra = 180 deg. "
+		"This property can be changed only when the map is not in the "
+		"ringsparse representation, and may improve ringsparse memory use.")
```

Comment:
Maybe add some more text here: "Although this only affects the map when using ring sparse, this property can only be..."

Reply:
Removed the text altogether